src/test/java/com/dougnoel/sentinel/pages/PageManagerTests.java

```diff
@@ -31,4 +35,26 @@ public void getWindowHandleInSameWindow() throws InterruptedException {
 			//suppress
 		}
 	}
+	
+	@Test
+	public void checkBackgroundColorOfPage() {
```

Comment:
Can these tests cover the same code as the ones in Element tests? What's the minimum amount of tests we can do to get code coverage? And if the answer is no, can you refactor the code so these two objects use the same code?

Reply:
Done